lib/html_parser.dart

```diff
@@ -660,7 +660,7 @@ class HtmlParser extends StatelessWidget {
     List<StyledElement> toRemove = new List<StyledElement>();
     bool lastChildBlock = true;
     tree.children?.forEach((child) {
-      if (child is EmptyContentElement) {
+      if (child is EmptyContentElement || child is EmptyLayoutElement) {
```

Comment:
@erickok I think this is what I'm most interested in. Whether or not there should be a `EmptyLayoutElement` or if there is a better solution here. I'm wondering if the super class of `LayoutElement` is incorrect for this reason

Reply:
Yeah, the `<details>` tag being a LayoutElement sounds totally legit to me.

Reply:
Both `EmptyContentElement` and `EmptyLayoutElement`'s super-superclass is `StyledElement`. 
Technically what you could do is make `EmptyContentElement` extend `StyledElement` and generalize the `parseReplacedElement` and `parseLayoutElement` functions to return a `StyledElement` instead of `ReplacedElement` and `LayoutElement`, respectively. Don't know if this is an option you would consider though.